tests/func/011.phpt

```diff
@@ -1,10 +1,10 @@
 --TEST--
 Test bitwise AND, OR, XOR, NOT and logical NOT in INI via error_reporting
 --INI--
-error_reporting = E_ALL & E_NOTICE | E_STRICT ^ E_DEPRECATED & ~E_WARNING | !E_ERROR
+error_reporting = E_ALL & E_NOTICE ^ E_DEPRECATED & ~E_WARNING | !E_ERROR
```

Comment:
Should probably be replaced with something else like E_WARNING, this is a parsing test. Similar below.